cluster/gce/configure-vm.sh

```diff
@@ -242,13 +246,20 @@ EOF
 
 # This should only happen on cluster initialization
 function create-salt-auth() {
-  mkdir -p /srv/salt-overlay/salt/nginx
-  echo "${MASTER_HTPASSWD}" > /srv/salt-overlay/salt/nginx/htpasswd
+  local -r htpasswd_file="/srv/salt-overlay/salt/nginx/htpasswd"
+
+  if [ ! -e ${htpasswd_file} ]; then
+    mkdir -p /srv/salt-overlay/salt/nginx
+    echo "${MASTER_HTPASSWD}" > "${htpasswd_file}"
+  fi
+
+  if [ -e "${KNOWN_TOKENS_FILE}" ]; then
```

Comment:
I debated it, too, and had it both ways. I have a strong preference for column 0, as you notice, but I'm happy to flip it back.